lib/ui/form/widget/form_survey_question_page.dart

```diff
@@ -1,13 +1,57 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_survey/theme/app_colors.dart';
+import 'package:flutter_survey/theme/app_dimensions.dart';
 
 class FormSurveyQuestionPage extends StatelessWidget {
+  final int questionIndex;
+  final int questionTotal;
+
   const FormSurveyQuestionPage({
     Key? key,
+    required this.questionIndex,
+    required this.questionTotal,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    // TODO: Add UI & integrate question page #24 & #25
-    return const Text("Questions & Answers");
+    return SafeArea(
+      child: Padding(
+        padding: const EdgeInsets.symmetric(
+          vertical: AppDimensions.spacing54,
+          horizontal: AppDimensions.spacing16,
+        ),
+        child: Column(
+          children: [
+            _buildQuestionCounter(context),
+            const SizedBox(height: AppDimensions.spacing8),
+            _buildQuestion(context),
```

Comment:
If we put the Next button outside of this page's content. How can we ensure the answer form to not overlap the Next button? 🤓

Reply:
I think I can add `Padding` at the bottom once it happens, what do you think? 🙏

Reply:
It could be a valid way. Let's see what we can do with it 🚀